DataConnectors/Qualys VM/host.json

```diff
@@ -0,0 +1,11 @@
+{
+    "version": "2.0",
+    "functionTimeout": "00:10:00",
+    "managedDependency": {
+      "Enabled": true
+    },
+    "extensionBundle": {
+      "id": "Microsoft.Azure.Functions.ExtensionBundle",
+      "version": "[1.*, 2.0.0)"
```

Comment:
The opening bracket [  and closing parenthesis  ) - are not matching

Reply:
This is taken from the sample from the MFST documentation and also compared against out data connectors (Zoom) 

https://docs.microsoft.com/en-us/azure/azure-functions/functions-host-json